crypto/evp/p_lib.c

```diff
@@ -1411,6 +1415,12 @@ EVP_PKEY *EVP_PKEY_new(void)
     if (!evp_pkey_reset_unlocked(ret))
         goto err;
 
+    ret->lock = CRYPTO_THREAD_lock_new();
+    if (ret->lock == NULL) {
+        EVPerr(0, ERR_R_MALLOC_FAILURE);
```

Comment:
`ERR_raise(ERR_LIB_EVP, ERR_R_MALLOC_FAILURE);`

Please

Reply:
Thanks, fixed this now.